packages/builders/src/jest/jest.builder.ts

```diff
@@ -16,37 +17,66 @@ try {
 const { runCLI } = require('jest');
 
 export interface JestBuilderOptions {
+  codeCoverage?: boolean;
   jestConfig: string;
+  testDirectory?: string;
```

Comment:
Why is this necessary?

Reply:
vscode-jest passes in the file name only, without the full path.
To find out which file exactly that is, and to which project it belongs we need to know the directory.